deploy/wiremock/wiremock-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wiremock
+  namespace: REPLACE_NAMESPACE
+spec:
+  selector:
+    matchLabels:
+      app: wiremock
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wiremock
+    spec:
+      containers:
+        - name: wiremock
+          image: rodolpheche/wiremock
+          imagePullPolicy: "IfNotPresent"
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - mountPath: /home/wiremock
+              name: wiremock
+      volumes:
```

Comment:
Why does wiremock need a persistent volume?  Isn't it all ephemeral?

Reply:
right, we actually don't use any static stubs, so can remove the volume.